### PR TITLE
Support download dataset from origin URL in standalone mode

### PR DESCRIPTION
**Pull request by lidongze0629**
## Related issue number

related to #773

### CODE REVIEW

File: coordinator/gscoordinator/utils.py [hunk 1]
```diff
@@ -88,7 +88,15 @@
...
-    GRAPHSCOPE_HOME = os.path.join(COORDINATOR_HOME, "..")
+    if os.path.isdir(os.path.join(COORDINATOR_HOME, "graphscope.runtime")):
+        # install graphscope by bdist wheel
+        GRAPHSCOPE_HOME = os.path.join(COORDINATOR_HOME, "graphscope.runtime")
+    elif os.path.isdir(os.path.join(COORDINATOR_HOME, "..", "graphscope.runtime")):
+        # install graphscope with egg package:
+        GRAPHSCOPE_HOME = os.path.join(COORDINATOR_HOME, "..", "graphscope.runtime")
+    else:
+        # source code
+        GRAPHSCOPE_HOME = os.path.join(COORDINATOR_HOME, "..")
```
Comment (sighingnow) [inline coordinator/gscoordinator/utils.py:100]: Rewrittiing this part like

```python
# resolve from env
GRAPHSCOPE_HOME = os.environ.get("GRAPHSCOPE_HOME", None)

# resolve from pip installed package
if GRAPHSCOPE_HOME is None:
    if os.path.isdir(os.path.join(COORDINATOR_HOME, "graphscope.runtime")):
        GRAPHSCOPE_HOME = ...

# resolve from egg installed package
if GRAPHSCOPE_HOME is None:
    if os.path.isdir(os.path.join(COORDINATOR_HOME, "..", "graphscope.runtime")):
        ...

# resolve from develop source tree
if GRAPHSCOPE_HOME is None:
    GRAPHSCOPE_HOME = os.path.join(COORDINATOR_HOME, "..")
```

would be better, IMO.

File: python/graphscope/dataset/io_utils.py [hunk 2]
```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Alibaba Group Holding Limited. All Rights Reserved.
```
Comment (sighingnow) [inline python/graphscope/dataset/io_utils.py:4]: `Copyright 2020-2021`.

File: python/graphscope/dataset/ldbc.py [hunk 3]
```diff
@@ -270,17 +271,53 @@ def load_ldbc(sess, prefix, directed=True):
...
+        >>> g = load_ldbc(sess, "/path/to/dataset", True)
+
     """
-    prefix = os.path.expandvars(prefix)
+    if prefix is not None:
+        prefix = os.path.expandvars(prefix)
+    else:
+        fname = "ldbc_sample.tar.gz"
+        origin = (
+            "https://graphscope.oss-cn-beijing.aliyuncs.com/dataset/ldbc_sample.tar.gz"
```
Comment (sighingnow) [inline python/graphscope/dataset/ldbc.py:310]: We could add a custom domain name for the dataset, e.g., `https://data.graphscope.io`, you may find the DNS settings on aliyun webpage to see how to set custom domain name for oss buckets.

File: python/graphscope/dataset/modern_graph.py [hunk 4]
```diff
@@ -18,25 +18,57 @@
...
+        >>> sess = graphscope.session(mode="eager")
+        >>> g = load_modern_graph(sess, "/path/to/dataset", True)
     """
-    prefix = os.path.expandvars(prefix)
+    if prefix is not None:
+        prefix = os.path.expandvars(prefix)
+    else:
+        fname = "modern_graph.tar.gz"
+        origin = (
+            "https://graphscope.oss-cn-beijing.aliyuncs.com/dataset/modern_graph.tar.gz"
```
Comment (sighingnow) [inline python/graphscope/dataset/modern_graph.py:61]: Leave the `DATA_SITE` a global constant in the package.

File: python/requirements.txt [hunk 5]
```diff
@@ -7,6 +7,7 @@ nest_asyncio
 networkx
 numpy
 pandas
+progressbar
```
Comment (sighingnow) [inline python/requirements.txt:10]: Will tqdm be simpler?

Using `progressbar` is also OK to me.

**Review by sighingnow — APPROVED** [review]
LGTM.

Just some minor comments.

File: python/requirements.txt (hunk 5 above)
Comment (yecol) [reply python/requirements.txt:10]: I agree to use `tqdm`

**Pull request merged**